cmd/cmd_sync.go

```diff
@@ -7,13 +7,14 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-var synctargetCmd = &cli.Command{
-	Name:  "sync",
-	Usage: "Replicate data in a WebhookDB table into another database.",
+var dbsyncCmd = &cli.Command{
```

Comment:
Would it be hard to unify this code like we do the endpoints? We can pair on it.